### DISCUSSION TITLE
How to handle software without installers?

### DISCUSSION THREAD

**Discussion by roblillack** [microsoft/winget-cli#1256] [Q&A] [answered]
Hey there,

from reading the manifest spec I don't seem to be able to understand how to handle packages which are delivered as ready-to-use, self-contained binaries instead of installers (see https://github.com/roblillack/tack/releases/tag/v1.2.0 for example). InstallerType `exe` doesn't not seem to be the right thing to use.

Is this even possible?

Thanks,
—rob.

**Comment by denelon** [comment, marked as answer]
I think you are looking for https://github.com/microsoft/winget-cli/issues/182. We don't support standalone / portable executables yet.
